pkg/controller/storagecluster/uninstall_reconciler_test.go

```diff
@@ -246,3 +246,71 @@ func addFakeNodeAffinityKeyOnNodesAndSC(t *testing.T, reconciler ReconcileStorag
 		assert.NoError(t, err)
 	}
 }
+
+func TestDeleteNodeTaint(t *testing.T) {
+	testList := []struct {
+		createDefaultNodeTaint bool
+	}{
+		// case1: verify deleteNodeTaint deletes the default NodeTaint
+		{true},
+		// case2: verify does not get error out when node taints does not exist
+		{false},
+	}
```

Comment:
As in https://github.com/openshift/ocs-operator/pull/794#discussion_r505915347 , make this a bit more readable.

Reply:
Done